doc/user_guide/metadata_structure.rst

```diff
@@ -208,7 +246,17 @@ dwell_time
 exposure
     type: Float
 
-    The exposure time in seconds. This is relevant for TEM acquistion.
+    The exposure time in seconds. This is relevant for TEM acquisition.
+
+frame_number
+    type: int
+
+    The number of frames/spectra integrated during the acquisition.
```

Comment:
I don't really understand the question.
As it is, dwell time is the total integration time of the spectrum and in case several spectra have been cumulated during the acquisition, the frame_number is saved here. Not sure, it's very clear here though! Maybe I should just replace the word 'frame' with something else.